packages/paste-core/components/callout/src/Callout.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {NeutralIcon} from '@twilio-paste/icons/esm/NeutralIcon';
+import {NewIcon} from '@twilio-paste/icons/esm/NewIcon';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+import type {BoxStyleProps, BoxProps} from '@twilio-paste/box';
+import type {GenericIconProps} from '@twilio-paste/icons/src/types';
+import {isMarginTokenProp} from '@twilio-paste/style-props';
+
+type CalloutVariants = 'neutral' | 'warning' | 'error' | 'success' | 'new';
+
+export interface CalloutProps extends Partial<Omit<HTMLHeadingElement, 'children'>> {
```

Comment:
question: I see here you are extending the `HTMLHeadingElement` type for the Callout, was that intentional?

Reply:
nope, was supposed to be HTMLDivElement